2022-11-24T15:15:33.706216+00:00 app[web.1]: File "/app/.heroku/python/lib/python3.10/importlib/__init__.py", line 126, in import_module
2022-11-24T15:15:33.706216+00:00 app[web.1]: return _bootstrap._gcd_import(name[level:], package, level)
2022-11-24T15:15:33.706217+00:00 app[web.1]: File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
2022-11-24T15:15:33.706217+00:00 app[web.1]: File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
2022-11-24T15:15:33.706217+00:00 app[web.1]: File "<frozen importlib._bootstrap>", line 992, in _find_and_load_unlocked
2022-11-24T15:15:33.706217+00:00 app[web.1]: File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
2022-11-24T15:15:33.706218+00:00 app[web.1]: File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
2022-11-24T15:15:33.706218+00:00 app[web.1]: File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
2022-11-24T15:15:33.706218+00:00 app[web.1]: File "<frozen importlib._bootstrap>", line 992, in _find_and_load_unlocked
2022-11-24T15:15:33.706218+00:00 app[web.1]: File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
2022-11-24T15:15:33.706219+00:00 app[web.1]: File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
2022-11-24T15:15:33.706219+00:00 app[web.1]: File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
2022-11-24T15:15:33.706219+00:00 app[web.1]: File "<frozen importlib._bootstrap>", line 1004, in _find_and_load_unlocked
2022-11-24T15:15:33.706219+00:00 app[web.1]: ModuleNotFoundError: No module named 'uvicron'
2022-11-24T15:15:33.707343+00:00 app[web.1]: [2022-11-24 15:15:33 +0000] [9] [INFO] Worker exiting (pid: 9)
2022-11-24T15:15:33.735906+00:00 app[web.1]: [2022-11-24 15:15:33 +0000] [10] [INFO] Booting worker with pid: 10
2022-11-24T15:15:33.742726+00:00 app[web.1]: [2022-11-24 15:15:33 +0000] [10] [ERROR] Exception in worker process
2022-11-24T15:15:33.742727+00:00 app[web.1]: Traceback (most recent call last):
2022-11-24T15:15:33.742728+00:00 app[web.1]: File "/app/.heroku/python/lib/python3.10/site-packages/gunicorn/arbiter.py", line 589, in spawn_worker
2022-11-24T15:15:33.742729+00:00 app[web.1]: worker.init_process()
2022-11-24T15:15:33.742729+00:00 app[web.1]: File "/app/.heroku/python/lib/python3.10/site-packages/gunicorn/workers/base.py", line 134, in init_process
2022-11-24T15:15:33.742730+00:00 app[web.1]: self.load_wsgi()
2022-11-24T15:15:33.742730+00:00 app[web.1]: File "/app/.heroku/python/lib/python3.10/site-packages/gunicorn/workers/base.py", line 146, in load_wsgi
2022-11-24T15:15:33.742730+00:00 app[web.1]: self.wsgi = self.app.wsgi()
2022-11-24T15:15:33.742731+00:00 app[web.1]: File "/app/.heroku/python/lib/python3.10/site-packages/gunicorn/app/base.py", line 67, in wsgi
2022-11-24T15:15:33.742731+00:00 app[web.1]: self.callable = self.load()
2022-11-24T15:15:33.742731+00:00 app[web.1]: File "/app/.heroku/python/lib/python3.10/site-packages/gunicorn/app/wsgiapp.py", line 58, in load
2022-11-24T15:15:33.742732+00:00 app[web.1]: return self.load_wsgiapp()
2022-11-24T15:15:33.742732+00:00 app[web.1]: File "/app/.heroku/python/lib/python3.10/site-packages/gunicorn/app/wsgiapp.py", line 48, in load_wsgiapp
2022-11-24T15:15:33.742732+00:00 app[web.1]: return util.import_app(self.app_uri)
2022-11-24T15:15:33.742733+00:00 app[web.1]: File "/app/.heroku/python/lib/python3.10/site-packages/gunicorn/util.py", line 359, in import_app
2022-11-24T15:15:33.742733+00:00 app[web.1]: mod = importlib.import_module(module)
2022-11-24T15:15:33.742734+00:00 app[web.1]: File "/app/.heroku/python/lib/python3.10/importlib/__init__.py", line 126, in import_module
2022-11-24T15:15:33.742734+00:00 app[web.1]: return _bootstrap._gcd_import(name[level:], package, level)
2022-11-24T15:15:33.742735+00:00 app[web.1]: File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
2022-11-24T15:15:33.742735+00:00 app[web.1]: File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
2022-11-24T15:15:33.742735+00:00 app[web.1]: File "<frozen importlib._bootstrap>", line 992, in _find_and_load_unlocked
2022-11-24T15:15:33.742736+00:00 app[web.1]: File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
2022-11-24T15:15:33.742736+00:00 app[web.1]: File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
2022-11-24T15:15:33.742737+00:00 app[web.1]: File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
2022-11-24T15:15:33.742737+00:00 app[web.1]: File "<frozen importlib._bootstrap>", line 992, in _find_and_load_unlocked
2022-11-24T15:15:33.742737+00:00 app[web.1]: File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
2022-11-24T15:15:33.742738+00:00 app[web.1]: File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
2022-11-24T15:15:33.742738+00:00 app[web.1]: File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
2022-11-24T15:15:33.742738+00:00 app[web.1]: File "<frozen importlib._bootstrap>", line 1004, in _find_and_load_unlocked
2022-11-24T15:15:33.742738+00:00 app[web.1]: ModuleNotFoundError: No module named 'uvicron'
2022-11-24T15:15:33.742938+00:00 app[web.1]: [2022-11-24 15:15:33 +0000] [10] [INFO] Worker exiting (pid: 10)
2022-11-24T15:15:33.767048+00:00 app[web.1]: [2022-11-24 15:15:33 +0000] [4] [WARNING] Worker with pid 10 was terminated due to signal 15
2022-11-24T15:15:33.868116+00:00 app[web.1]: [2022-11-24 15:15:33 +0000] [4] [INFO] Shutting down: Master
2022-11-24T15:15:33.872877+00:00 app[web.1]: [2022-11-24 15:15:33 +0000] [4] [INFO] Reason: Worker failed to boot.
2022-11-24T15:15:34.069437+00:00 heroku[web.1]: Process exited with status 3
2022-11-24T15:15:34.125002+00:00 heroku[web.1]: State changed from starting to crashed
2022-11-24T15:15:34.128079+00:00 heroku[web.1]: State changed from crashed to starting
2022-11-24T15:15:41.679435+00:00 heroku[web.1]: Starting process with command `gunicorn -w 2 uvicron.workers.UvicornWorker mlapi.py:app`
2022-11-24T15:15:42.750062+00:00 app[web.1]: [2022-11-24 15:15:42 +0000] [4] [INFO] Starting gunicorn 20.1.0
2022-11-24T15:15:42.750361+00:00 app[web.1]: [2022-11-24 15:15:42 +0000] [4] [INFO] Listening at: http://0.0.0.0:10866 (4)
2022-11-24T15:15:42.750402+00:00 app[web.1]: [2022-11-24 15:15:42 +0000] [4] [INFO] Using worker: sync
2022-11-24T15:15:42.753741+00:00 app[web.1]: [2022-11-24 15:15:42 +0000] [9] [INFO] Booting worker with pid: 9
2022-11-24T15:15:42.756337+00:00 app[web.1]: [2022-11-24 15:15:42 +0000] [9] [ERROR] Exception in worker process
2022-11-24T15:15:42.756338+00:00 app[web.1]: Traceback (most recent call last):
2022-11-24T15:15:42.756350+00:00 app[web.1]: File "/app/.heroku/python/lib/python3.10/site-packages/gunicorn/arbiter.py", line 589, in spawn_worker
2022-11-24T15:15:42.756351+00:00 app[web.1]: worker.init_process()
2022-11-24T15:15:42.756351+00:00 app[web.1]: File "/app/.heroku/python/lib/python3.10/site-packages/gunicorn/workers/base.py", line 134, in init_process
2022-11-24T15:15:42.756352+00:00 app[web.1]: self.load_wsgi()
2022-11-24T15:15:42.756352+00:00 app[web.1]: File "/app/.heroku/python/lib/python3.10/site-packages/gunicorn/workers/base.py", line 146, in load_wsgi
2022-11-24T15:15:42.756352+00:00 app[web.1]: self.wsgi = self.app.wsgi()
2022-11-24T15:15:42.756352+00:00 app[web.1]: File "/app/.heroku/python/lib/python3.10/site-packages/gunicorn/app/base.py", line 67, in wsgi
2022-11-24T15:15:42.756353+00:00 app[web.1]: self.callable = self.load()
2022-11-24T15:15:42.756353+00:00 app[web.1]: File "/app/.heroku/python/lib/python3.10/site-packages/gunicorn/app/wsgiapp.py", line 58, in load
2022-11-24T15:15:42.756353+00:00 app[web.1]: return self.load_wsgiapp()
2022-11-24T15:15:42.756353+00:00 app[web.1]: File "/app/.heroku/python/lib/python3.10/site-packages/gunicorn/app/wsgiapp.py", line 48, in load_wsgiapp
2022-11-24T15:15:42.756353+00:00 app[web.1]: return util.import_app(self.app_uri)
2022-11-24T15:15:42.756354+00:00 app[web.1]: File "/app/.heroku/python/lib/python3.10/site-packages/gunicorn/util.py", line 359, in import_app
2022-11-24T15:15:42.756354+00:00 app[web.1]: mod = importlib.import_module(module)
2022-11-24T15:15:42.756354+00:00 app[web.1]: File "/app/.heroku/python/lib/python3.10/importlib/__init__.py", line 126, in import_module
2022-11-24T15:15:42.756354+00:00 app[web.1]: return _bootstrap._gcd_import(name[level:], package, level)
2022-11-24T15:15:42.756355+00:00 app[web.1]: File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
2022-11-24T15:15:42.756355+00:00 app[web.1]: File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
2022-11-24T15:15:42.756356+00:00 app[web.1]: File "<frozen importlib._bootstrap>", line 992, in _find_and_load_unlocked
2022-11-24T15:15:42.756356+00:00 app[web.1]: File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
2022-11-24T15:15:42.756356+00:00 app[web.1]: File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
2022-11-24T15:15:42.756356+00:00 app[web.1]: File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
2022-11-24T15:15:42.756357+00:00 app[web.1]: File "<frozen importlib._bootstrap>", line 992, in _find_and_load_unlocked
2022-11-24T15:15:42.756357+00:00 app[web.1]: File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
2022-11-24T15:15:42.756357+00:00 app[web.1]: File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
2022-11-24T15:15:42.756357+00:00 app[web.1]: File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
2022-11-24T15:15:42.756358+00:00 app[web.1]: File "<frozen importlib._bootstrap>", line 1004, in _find_and_load_unlocked
2022-11-24T15:15:42.756358+00:00 app[web.1]: ModuleNotFoundError: No module named 'uvicron'
2022-11-24T15:15:42.756408+00:00 app[web.1]: [2022-11-24 15:15:42 +0000] [9] [INFO] Worker exiting (pid: 9)
2022-11-24T15:15:42.788906+00:00 app[web.1]: [2022-11-24 15:15:42 +0000] [4] [INFO] Shutting down: Master
2022-11-24T15:15:42.788937+00:00 app[web.1]: [2022-11-24 15:15:42 +0000] [4] [INFO] Reason: Worker failed to boot.
2022-11-24T15:15:42.953579+00:00 heroku[web.1]: Process exited with status 3
2022-11-24T15:15:43.098156+00:00 heroku[web.1]: State changed from starting to crashed
2022-11-24T15:15:43.970450+00:00 heroku[router]: at=error code=H10 desc="App crashed" method=GET path="/" host=salty-sierra-33429.herokuapp.com request_id=9f3387c1-6c00-4504-877f-f12ccfb9ea59 fwd="197.1.8.90" dyno= connect= service= status=503 bytes= protocol=https
2022-11-24T15:15:46.318610+00:00 heroku[router]: at=error code=H10 desc="App crashed" method=GET path="/favicon.ico" host=salty-sierra-33429.herokuapp.com request_id=cf9e8f1e-7d7b-44b9-8044-3c7bac60f241 fwd="197.1.8.90" dyno= connect= service= status=503 bytes= protocol=https
2022-11-24T15:15:47.000000+00:00 app[api]: Build succeeded
2022-11-24T15:15:58.870942+00:00 heroku[router]: at=error code=H10 desc="App crashed" method=GET path="/" host=salty-sierra-33429.herokuapp.com request_id=f76b7b46-0610-4f91-87b9-2f9a9871a2fd fwd="197.1.8.90" dyno= connect= service= status=503 bytes= protocol=https
2022-11-24T15:16:01.118695+00:00 heroku[router]: at=error code=H10 desc="App crashed" method=GET path="/favicon.ico" host=salty-sierra-33429.herokuapp.com request_id=2434075c-d249-4a0f-8b73-ce768fe80760 fwd="197.1.8.90" dyno= connect= service= status=503 bytes= protocol=https
2022-11-24T15:16:07.138508+00:00 heroku[router]: at=error code=H10 desc="App crashed" method=GET path="/" host=salty-sierra-33429.herokuapp.com request_id=ee6a089c-c1c7-42f3-9f91-90d97ad309ae fwd="197.1.8.90" dyno= connect= service= status=503 bytes= protocol=https
2022-11-24T15:16:10.850849+00:00 heroku[router]: at=error code=H10 desc="App crashed" method=GET path="/favicon.ico" host=salty-sierra-33429.herokuapp.com request_id=bbb5be5a-8359-40d6-a802-545a22cb3c61 fwd="197.1.8.90" dyno= connect= service= status=503 bytes= protocol=https
